enterprise/frontend/src/embedding-sdk/types/theme/index.ts

```diff
@@ -23,21 +25,7 @@ export interface MetabaseTheme {
   components?: MetabaseComponentTheme;
 }
 
-export interface MetabaseColors {
-  /** Primary brand color */
-  brand?: string;
-
-  /** Text color on dark elements. Should be a lighter color for readability. */
-  "text-dark"?: string;
-
-  /** Text color on light elements. Should be a darker color for readability. */
-  "text-light"?: string;
-
-  /** Lighter variation of dark text on light elements. */
-  "text-medium"?: string;
-}
-
-export type MetabaseColor = keyof MetabaseColors;
+export type MetabaseColors = ColorPalette;
```

Comment:
@deniskaber we have a plan to remap the external-facing color names to the internal ones in https://github.com/metabase/metabase/issues/42680, as the names we use currently (e.g. `text-dark` and `text-light`) does not make sense for SDK customers who uses dark mode, so the idea there is to remap them (e.g. `text-primary` to `text-dark`, `text-secondary` to `text-light` and so on) - [see the discussion here](https://www.notion.so/metabase/Next-Theming-Milestones-4e82017c97844adb983add93fed2c6d1?pvs=4#23e69b29b3ea42698ca087b01d5c904a)

If we don't remap the color names, then we would have to do a rename for the entire codebase - which is going to be a lot of diffs (and potentially cause confusion internally)